### PR TITLE
Style for the table view

### PR DESCRIPTION
**Pull request by alex-krasn**
I adapted table style to be more align with overall app design:
-add horizontal padding for cells content
- make borders less prominent - darker and bit bolder (for readability )
- moved to the left and spaced out scrollbar (it's not touching  table anymore)
- add some space to close button from the edges (too close in my opinion)
...and fix some redundant  code and misspelling
After:
Screen Shot 2020-05-22 at 1 14 45 PM
 

Before: 
Screen Shot 2020-05-22 at 1 36 14 PM

### CODE REVIEW

File: src/react/components/pages/editorPage/tableView.scss [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-    border: 1.2px solid rgba(255,255,255, 0.9);
-    td {
-        border: 0.6px solid rgba(255,255,255, 0.7);
-        min-width: 80px;
-        padding: 8px 16px 8px 8px;
-    }
-    td:empty::after {
-        content: "\00a0";
+.table-view-container {
+    direction: rtl;
```
Comment (stew-ro) [inline src/react/components/pages/editorPage/tableView.scss:5]: why make scroll bar on left side? Typically they're on the right side, right?

Comment (alex-krasn) [reply src/react/components/pages/editorPage/tableView.scss:5]: So it's not too close to table and close btn. I think, this way it looks more balanced.

Comment (alex-krasn) [reply src/react/components/pages/editorPage/tableView.scss:5]: It can be on another side it's really does not matter much to me.

Comment (tianxiangs) [reply src/react/components/pages/editorPage/tableView.scss:5]: I think the balanced style looks good, but maybe should still put it on the right, more follows the common sense.

Comment (stew-ro) [reply src/react/components/pages/editorPage/tableView.scss:5]: Adding margin to the table adds some separation between the table border and the scrollbar:
image
maybe we should do that instead

Comment (alex-krasn) [reply src/react/components/pages/editorPage/tableView.scss:5]: yeah, I'm moving it back.

Comment (alex-krasn) [reply src/react/components/pages/editorPage/tableView.scss:5]: Screen Shot 2020-05-22 at 2 49 27 PM

**Comment by alex-krasn** [comment]
I meant to the right, sorry :-)

**Pull request merged**